src/main/webapp/static/academic/css/dataTables.responsive.css

```diff
@@ -0,0 +1,94 @@
+table.dataTable.dtr-inline.collapsed tbody td:first-child,
```

Comment:
dataTables is already responsive via integration with bootstrap: https://github.com/FenixEdu/fenixedu-academic/blob/master/src/main/webapp/javaScript/dataTables/media/js/jquery.dataTables.bootstrap.min.js
Why are you adding another dependency?